README.md

```diff
@@ -11,3 +11,16 @@ _**Prerequisits: This repository requires node v16 and up**_
 1. Clone this repository
 2. Install dependecies with `npm i`
 3. Test and lint with `npm test`
+
+## Deploy Local Commits
+
+### The first time you run a deploy
+
+To be able to run the netlify commands locally you will need to run `npx netlify login` once after `npm install` to authenticate and obtain a token.
+
+### Every time when doing a deploy
+
+To create a deploy of the current commit, run `npm run deployCommit`. This will create a build in the `./dist` folder and create a manual deployment to a unique address that will be generated at the time of deploy. 
+To deploy to [https://new-lexeme-special-page.netlify.app](https://new-lexeme-special-page.netlify.app) run `npx netlify deploy --prod`.
```

Comment:
Shouldn’t this command also have `--dir='./dist'`? (Or maybe `npm run deployCommit --prod` would also work?)

Reply:
also the current command works.

Reply:
with the current state of the deployCommit script. `npm run deployCommit --prod` doesn't deploy to production.

Reply:
it deploys by default to `./dist`. i dont understand the need of adding it to three places now.

Reply:
That’s confusing, does it not ask about the directory in that case? (I haven’t tried it out yet.)

Reply:
Here’s what I get:

```
$ npx netlify deploy --prod
Please provide a publish directory (e.g. "public" or "dist" or "."):
/var/www/html/wiki1/extensions/WikibaseLexeme/resources/special/new-lexeme
? Publish directory (.) 
```

Reply:
Maybe it remembers your choice, and that’s why, on your end, it deploys `./dist` by default?

Reply:
The directory is specified once when you login for the first time. You won't be asked for the directory again.

Reply:
I do have an account, it made me create one. I don’t think it asked me about this setting at any point.

Reply:
> with the current code of the deployCommit script. `npm run deployCommit --prod` doesn't deploy to production.

My bad, needs two more dashes. `npm run deployCommit -- --prod` seems to do the right thing (but I get an error, because my personal account isn’t allowed to deploy to this site, of course).

Reply:
I would suggest removing this instruction altogether, since we will most probably not need to deploy to any production site. We only want commit deployments. Our "prod" is WikibaseLexeme itself.